precalculate-measurements.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	measuredboot "github.com/edgelesssys/uplosi/measured-boot"
+	"github.com/edgelesssys/uplosi/measured-boot/measure"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newPrecalculateMeasurementsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "precalculate-measurements <image>",
```

Comment:
```suggestion
		Use:   "measurements <image>",
```
nit: just using `uplosi measurements <image>` sounds a lot better